crates/anvil/tests/it/eip4844.rs

```diff
@@ -204,3 +204,45 @@ async fn can_check_blob_fields_on_genesis() {
     assert_eq!(block.header.blob_gas_used, Some(0));
     assert_eq!(block.header.excess_blob_gas, Some(0));
 }
+
+#[tokio::test(flavor = "multi_thread")]
+async fn can_correctly_estimate_blob_gas_with_recommended_fillers() {
+    let node_config = NodeConfig::test().with_hardfork(Some(EthereumHardfork::Cancun.into()));
+    let (_api, handle) = spawn(node_config).await;
+
+    let provider = http_provider(&handle.http_endpoint());
+
+    let accounts = provider.get_accounts().await.unwrap();
+    let alice = accounts[0];
+    let bob = accounts[1];
+
+    let sidecar: SidecarBuilder<SimpleCoder> = SidecarBuilder::from_slice(b"Blobs are fun!");
+    let sidecar = sidecar.build().unwrap();
+
+    let tx = TransactionRequest::default().with_to(bob).with_blob_sidecar(sidecar);
+    let tx = WithOtherFields::new(tx);
+
+    // Send the transaction and wait for the broadcast.
+    let pending_tx = provider.send_transaction(tx).await.unwrap();
+
+    println!("Pending transaction... {}", pending_tx.tx_hash());
+
+    // Wait for the transaction to be included and get the receipt.
+    let receipt = pending_tx.get_receipt().await.unwrap();
+
+    // Grab the processed transaction.
+    let tx = provider.get_transaction_by_hash(receipt.transaction_hash).await.unwrap().unwrap();
+
+    println!(
+        "Transaction included in block {}",
+        receipt.block_number.expect("Failed to get block number")
+    );
+
+    assert!(tx.max_fee_per_blob_gas.unwrap() >= BLOB_TX_MIN_BLOB_GASPRICE);
```

Comment:
This currently will still fail until https://github.com/alloy-rs/alloy/pull/1389 has been merged as it will set the `max_fee_per_blob_gas` field correctly